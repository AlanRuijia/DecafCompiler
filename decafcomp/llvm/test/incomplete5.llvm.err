4: syntax error at char 21
; ModuleID = 'Test'
