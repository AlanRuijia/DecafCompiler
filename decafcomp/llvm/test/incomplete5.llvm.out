Syntax Error : at line 4 char 21
